Recrutify/Recrutify.Services/Services/ScheduleService.cs

```diff
@@ -36,6 +41,25 @@ public async Task<IEnumerable<ScheduleDTO>> GetByUserPrimarySkillAsync(Guid proj
             return _mapper.Map<List<ScheduleDTO>>(schedules);
         }
 
+        public async Task BulkAppointOrCancelInterviewsAsync(IEnumerable<InterviewAppointmentDTO> interviewAppointmentDTOs, Guid projectId)
+        {
+            var candidates = await _candidateService.GetCandidatesByIdsAsync(interviewAppointmentDTOs.Select(c => c.CandidateId));
+            var scheduleCandidateInfos = GetScheduleCandidateInfos(candidates, interviewAppointmentDTOs, projectId);
+            var appointInterviewSlots = interviewAppointmentDTOs.Select(a => new InterviewAppointmentSlotDTO()
```

Comment:
why you create InterviewAppointmentSlotDTO here, and then map it InterviewAppointmentSlot? 
and moreover that you map it twice! What is the sense to create DTO only for mapping it to model?